src/Infrastructure/Identity/ApplicationUser.cs

```diff
@@ -12,4 +13,6 @@ public class ApplicationUser : IdentityUser
     public DateTime RefreshTokenExpiryTime { get; set; }
 
     public string? ObjectId { get; set; }
+
+    public bool HasRootEmailAddress => Email == MultitenancyConstants.Root.EmailAddress;
```

Comment:
I wouldn't introduce a new property for this add migrations again.

Reply:
As I understand it, it would be a calculated property which is not in the database... Not sure if you have to add the `[NotMapped]` attribute or if that's handled automatically for read-only properties by ef core...

I'm not sure about the usefulness of `HasRootEmailAddress` here, but the `IsRootTenant` on `FSHTenantInfo` could prove to be more useful for sure! (as that functionality is used in a lot more places).